.github/workflows/release.yml

```diff
@@ -135,6 +135,13 @@ jobs:
     name: Build and push Semgrep Docker image
     runs-on: ubuntu-latest
     if: ${{ !contains(github.ref,'-test-release') }}
+    needs:
```

Comment:
Again a comment would be nice, to explain why we do that here (basically what you had in your PR comment, but
even though we can find back the info with git blame, I think it's better to put it in comment directly).

Reply:
Thanks for the comment - going to resolve as I want to get this in before @brandonspark releases.